packages/zosfiles/__tests__/__integration__/cli/copy/ds/__snapshots__/cli.files.copy.ds.integration.test.ts.snap

```diff
@@ -0,0 +1,115 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Copy Data Set should display the help 1`] = `
+"================Z/OS FILES COPY DATA SET HELP===============
+
+ COMMAND NAME
+ ------------
+
+   data-set | ds
+
+ DESCRIPTION
+ -----------
+
+   Copy a data set to another data set
+
+ USAGE
+ -----
+
+   zowe zos-files copy data-set <fromDataSetName> <toDataSetName> [options]
+
+ POSITIONAL ARGUMENTS
+ --------------------
+
+   fromDataSetName		 (string)
+
+      The name of the data set that you want to copy from
+
+   toDataSetName		 (string)
+
+      The name of the data set that you want to copy to
+
+ ZOSMF CONNECTION OPTIONS
+ ------------------------
+
+   --host  | -H (string)
+
+      The z/OSMF server host name.
+
+   --port  | -P (number)
+
+      The z/OSMF server port.
+
+      Default value: 443
+
+   --user  | -u (string)
+
+      Mainframe (z/OSMF) user name, which can be the same as your TSO login.
+
+   --password  | --pass | --pw (string)
+
+      Mainframe (z/OSMF) password, which can be the same as your TSO password.
+
+   --reject-unauthorized  | --ru (boolean)
+
+      Reject self-signed certificates.
+
+      Default value: true
+
+   --base-path  | --bp (string)
+
+      The base path for your API mediation layer instance. Specify this option to
+      prepend the base path to all z/OSMF resources when making REST requests. Do not
+      specify this option if you are not using an API mediation layer.
+
+ PROFILE OPTIONS
+ ---------------
+
+   --zosmf-profile  | --zosmf-p (string)
+
+      The name of a (zosmf) profile to load for this command execution.
+
+ GLOBAL OPTIONS
+ --------------
+
+   --response-format-json  | --rfj (boolean)
+
+      Produce JSON formatted data from a command
+
+   --help  | -h (boolean)
+
+      Display help text
+
+   --help-examples  (boolean)
+
+      Display examples for all the commands in a the group
+
+   --help-web  | --hw (boolean)
+
+      Display HTML help in browser
+
+ EXAMPLES
+ --------
+
+   - Copy the data set named 'USER.FROM.SET' to the data set
+   named 'USER.TO.SET':
+
+      $ zowe zos-files copy data-set \\"USER.FROM.SET\\" \\"USER.TO.SET\\"
+
+   - Copy the data set named 'USER.FROM.SET(MEM1)' to the data
```

Comment:
Should this say `Copy the data set *member* named 'USER.FROM.SET(MEM1)' to the data set *member* named 'USER.TO.SET(MEM2)'`